Dockerfile

```diff
@@ -13,6 +13,11 @@ ENV LD_LIBRARY_PATH /usr/local/lib
 RUN mkdir /freqtrade
 WORKDIR /freqtrade
 
+# Update PIP
+RUN python -m pip install --upgrade pip
+RUN pip install future
+RUN pip install numpy
```

Comment:
what exactly is future for ... ?
not saying it's not needed, just trying to understand

also - i would try to combine multiple consecutive `RUN` statements to one (using `\` and `&&`, as this keeps the image iterations lower, and therefore also the size.

and numpy is there, line 23 ... :)